server/src/analyzer/service.ts

```diff
@@ -69,15 +68,17 @@ export class AnalyzerService {
     private _analyzeTimer: any;
     private _requireTrackedFileUpdate = true;
     private _lastUserInteractionTime = Date.now();
+    private _configFileName: string;
 
-    constructor(instanceName: string, console?: ConsoleInterface) {
+    constructor(instanceName: string, console?: ConsoleInterface, configFileName?: string) {
```

Comment:
It's important for us to support both "pyrightconfig.json" and "mspythonconfig.json" in pyrx. Otherwise there will be no way for users to use the pyright command-line tool along with the Python extension (with pyrx). It looks like the current code supports only one or the other. I think it needs to support both. I'm fine if it looks first for "mspythonconfig.json" and falls back to "pyrightconfig.json".

Reply:
OK.